free-courses-id.md

```diff
@@ -193,6 +193,7 @@
 * [Belajar Laravel 7 Dari Awal](https://www.youtube.com/playlist?list=PLRKMmwY3-5MxfIKTn_wZ49XlplwHtz1AV) - Parsinta
 * [Membangun Web Profil Band dan Lirik dengan Laravel 8](https://www.youtube.com/playlist?list=PLRKMmwY3-5Mzoti-pT2MGuQERTd1_sm21) - Parsinta
 * [Tutorial Laravel 7.x Dasar](https://www.youtube.com/playlist?list=PLCZlgfAG0GXBucXejxeeqCe_NWZS-67q_) - Sekolah Koding
+* [Tutorial Laravel Bahasa Indonesia](https://id-laravel.com/) - ID Laravel
```

Comment:
```suggestion
* [Tutorial Laravel Bahasa Indonesia](https://id-laravel.com) - ID Laravel
```

Hello, simply removing the trailing "/" would process the Travis CI build correctly and you would be ready to merge your PR.

Check out the [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) file. Right under guidelines it mentions "on root domains, strip the trailing slash: http://example.com instead of http://example.com/"